proxy/invoke.go

```diff
@@ -147,7 +147,10 @@ func parseHeaders(headers []string) (map[string]string, error) {
 
 // validateMethod validates the HTTP request method
 func validateHTTPMethod(httpMethod string) error {
-	var allowedMethods = []string{http.MethodGet, http.MethodPost}
+	var allowedMethods = []string{
+		http.MethodGet, http.MethodPost, http.MethodPut, http.MethodPatch, http.MethodDelete,
```

Comment:
SGTM - let's just remove `Head` and `Options` and I'll merge for milestone: `0.7.2`.

Reply:
done :+1: